docs/walletkit/android/experimental/chain-abstraction.mdx

```diff
@@ -13,8 +13,7 @@ Apps need to pass `gas` as null, while sending a transaction to allow proper gas
 When sending a transaction, you need to:
 1. Check if the required chain has enough funds to complete the transaction
 2. If not, use the `prepare` method to generate necessary bridging transactions
-3. Check the status of the bridging transactions and wait for them to be completed
```

Comment:
i think it's worth to mention: "Sign routing and initial transaction"

Reply:
Updated